.editorconfig

```diff
@@ -0,0 +1,22 @@
+# top-most EditorConfig file
+root = true
+
+# Apply to all files
+[*]
+end_of_line = lf
```

Comment:
I'm not sure about that. I thought the standard those days (esp in macOS) was CRLF, not just LF? 🤔

Reply:
Actually, nvm, I checked and macOS (as well as Unix) uses `0x0A`, aka `LF` nowadays, so this is the right one after all.